test/convert/convertContext.test.ts

```diff
@@ -201,6 +201,7 @@ describe('Convert Transaction Constructs', () => {
             children: [
               {
                 name: labelsFileName,
+                // eslint-disable-next-line @typescript-eslint/no-unsafe-argument
```

Comment:
this feels like another rule that could be set to false. Looks let we have no-explicit-any in place already so if we're working with any it is expected.

Reply:
there's **non**-explicit any scenarios.